linter_exclusions.yml

```diff
@@ -19,6 +16,9 @@ dt endpoint create eventhub:
     eventhub_resource_group:
       rule_exclusions:
       - parameter_should_not_end_in_resource_group
+dt data-history connection create:
+  rule_exclusions:
+      - require_wait_command_if_no_wait
```

Comment:
We should be consistent with indentation (compare indentation between other rule_exclusions entities).